.pipelines/loc/loc.yml

```
@@ -35,6 +35,7 @@ steps:
      **\Resources.resx
      **\Resource.resx
      **\Resources.resw
+     -src\modules\MouseWithoutBorders\App\Properties\Resources.resx
```

okay Clint, try this one...

```suggestion
     !(src\modules\MouseWithoutBorders\App\Properties\Resources.resx)
```

---

If that doesn't work, you may need...

```suggestion
     !(src/modules/MouseWithoutBorders/App/Properties/Resources.resx)
```